Schools/_posts/2019-02-01-CMS-SHERPA.md

```diff
@@ -0,0 +1,9 @@
+---
+title: The First CMS SHERPA Workshop
+source: https://indico.cern.ch/event/779232/
```

Comment:
This is a restricted URL (probably only CMS people can see it). So for the HSF site you'd need to find some public URL that's useful or remove this entry.

Reply:
done